### ISSUE TITLE
Custom arrowheads

### ISSUE THREAD

**Issue by JakobThumm** [tikzit/tikzit#148]
I want to use custom arrowheads such as `-{Triangle[scale=2]}`.
Unfortunately, `tikzit` classifies an edge by the atoms `-`, `->`, or `-|`. So, changing the arrowhead changes the type from edge to node.
Is there some way to go around this?

**Comment by akissinger** [comment]
Yes. You can use the separate `<` and `>` properties to give custom arrowhead definitions to tex without interfering with tikzit's edge style logic, e.g.

    \tikzstyle{pointer edge}=[->, >={latex[scale=2]}]

Essentially, the only role picking `->` vs `-` etc plays here is telling tikzit how to display the edge in the GUI.

**Comment by JakobThumm** [comment]
Works like a charm. Thank you for addressing this issue!
